packages/shared/src/lib/board_types.ts

```diff
@@ -15,4 +15,7 @@ export type DefaultBoardConfig = {
 export type DefaultBoardState = {
   board: MulticolorStone[] | MulticolorStone[][];
   backgroundColor?: string;
+  // TODO: we should probably merge this into board
+  // score would be some field on MulticolorStone
+  score_board?: (string[] | null)[] | (string[] | null)[][];
```

Comment:
I forgot to pipe this through in some way.  Also added the `TODO` since the "shapes" of `board` and `score_board` are similar